src/client/CMakeLists.txt

```diff
@@ -17,4 +17,6 @@ add_library(client-lib
     rendering/RenderingController.cpp
 )
 
+add_subdirectory("extern/cxxopts")
```

Comment:
extern files shouldn't be in `src`. `src` is only for code we've written 
Mote this to the top level `extern` dir (where catch is)